File: Java/Estrutura repetitiva/Para/Sequencia_impares.java
import java.util.Locale;
import java.util.Scanner;

public class Sequencia_impares {
        public static void main(String[] args) {

        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        int x, i;

        System.out.print("Digite o valor de x: ");
        x = sc.nextInt();


        for (i = 1 ; i <= x; i++) {
            if (i % 2 != 0) {
                System.out.println(i);
            }
        }


        sc.close();

    }
}

File: C/Estrutura Repetitiva/Para/Sequencia_impares.c
#include <stdio.h>
#include <string.h>



int main ()
{
    int i, x, soma;

    printf ("Digite o valor de X: ");
    scanf ("%d", &x);

    for (i = 0; i < x; i++)
    {
        i = i + 1;
        if (i % 2 != 0);
        printf ("%d\n", i);
    }


    return 0;
}